components/compliance-service/ingest/projectupdater/manager.go

```diff
@@ -30,7 +30,7 @@ const (
 type Manager struct {
 	state                 string
 	projectUpdateID       string
-	esJobID               string
+	esJobIDs              []string
```

Comment:
Are we expecting more things to be added to this? Using a list for two things might be a bit of premature abstraction, IMO.

Reply:
I mean this in reference to other things introduced in this PR, not just this value specifically.

Reply:
Would you think it would be better to list out both types of job IDs like below? 
```
esReportJobID string
esSummaryJobID string
```

Reply:
I don't know of anything other jobs that are going to be added to this.

Reply:
That's what I was thinking, looping over a list that's only going to ever be two things seems a bit overkill.

Reply:
I am going to leave it with the slice of IDs for now. We are still working on this code, so we can find a better solution later.